modules/oe_showcase_list_pages/tests/src/ExistingSite/ListPagesTest.php

```diff
@@ -81,42 +111,100 @@ public function testCreateListPages() {
     $this->assertNotNull($title_input);
     $this->assertNotNull($publication_date_input);
 
-    // Filter results by date.
+    // Filter the News results by date.
     $publication_date_input->setValue('gt');
     $date_input = $filter_form->findField('Date');
     $date_input->setValue('2022-04-04');
     $search_button->click();
-    $this->assertSearchResultsTitle(8);
+    $this->assertSearchResultsTitle('News', 6);
     $this->assertSearchResults([
       'News number 4',
       'News number 5',
       'News number 6',
       'News number 7',
       'News number 8',
       'News number 9',
-      'News number 10',
-      'News number 11',
     ]);
 
-    // Filter results by title.
+    // Filter the News results by title.
     $title_input->setValue('News number 8');
     $search_button->click();
-    $this->assertSearchResultsTitle(1);
+    $this->assertSearchResultsTitle('News', 1);
```

Comment:
As suggested let's pass the entire title to the method.
```suggestion
    $this->assertSearchResultsTitle('News list page', 1);
```